### PR TITLE
Feature/ATG-60 Enums (figures colors)

### PR DESCRIPTION
**Pull request by IraCherginyshka**
Create colors enum for figures

### CODE REVIEW

File: src/app/enums/figures-colors.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const enum FiguresColors {
+  'Default' = '#2b3a42',
```
Comment (oleksandrstarov) [inline src/app/enums/figures-colors.ts]: No need to use string literals, i recommend to use DEFAULT='val'

File: src/app/enums/figures-colors.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const enum FiguresColors {
```
Comment (oleksandrstarov) [inline src/app/enums/figures-colors.ts]: should be declared as export enum FiguresColors

Comment (oleksandrstarov) [inline src/app/enums/figures-colors.ts]: Please rename to figures-colors.enum.ts

File: src/app/enums/figures-colors.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  'First' = '#fd0404',
+  'Second' = '#2e36d5',
+  'Third' = '#19af09',
+  'Fourth' = '#f8ab05',
+  'Fifth' = '#13e4e7',
+  'Sixth' = '#ff7300',
+  'Seventh' = '#d131c4',
+}
+
+export default FiguresColors;
```
Comment (oleksandrstarov) [inline src/app/enums/figures-colors.ts]: no need to use this line

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/app/enums/figures-colors.ts (hunk 1 above)
Comment (IraCherginyshka) [reply src/app/enums/figures-colors.ts]: Done

File: src/app/enums/figures-colors.ts (hunk 2 above)
Comment (IraCherginyshka) [reply src/app/enums/figures-colors.ts]: Done

Comment (IraCherginyshka) [reply src/app/enums/figures-colors.ts]: Done

File: src/app/enums/figures-colors.ts (hunk 3 above)
Comment (IraCherginyshka) [reply src/app/enums/figures-colors.ts]: Done

**Review by oleksandrstarov — APPROVED** [review]

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**